articles/basic-types-and-value-literals.html

```diff
@@ -119,7 +119,7 @@ <h3>Built-in Basic Types in Go</h3>
 for the built-in boolean and string types.
 So there can be many boolean and string types.
 The same is for any kinds of numeric types.
-The following are some type declarations examples.
+The following are some type of declarations examples.

```

Comment:
Here, it is really a mistake. But I think change it to "some type declaration examples" is better. For `type declaration` is a terminology.